src/main/resources/swagger/api-docs.yaml

```diff
@@ -4361,6 +4361,12 @@ definitions:
       expression:
         type: string
         description: Expression that resolves to one or more entities matching the entity type in the method configuration.
+      useCallCache:
```

Comment:
I noticed `useCallCache` was missing so I added it -- this should be in sync with Rawls now.